x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/whitelist/InternalSqlScriptUtils.java

```diff
@@ -146,27 +152,27 @@ public static Boolean regex(String value, String pattern) {
     //
     // Math
     //
-    public static Number add(Number left, Number right) {
```

Comment:
Why did you switch to object? We should have already validated they are Numbers or not?

Reply:
Ah, ignore, I rushed, it's because of numeric operations between intervals.